src/app/components/core/manage-services/list-manage-services.component.html

```diff
@@ -0,0 +1,99 @@
+                          (click)="setService(service);openModal('confirmation-dialog')">
+                          <span>Disable</span>
+                        </a>
+                        <a *ngIf="sharedService.checkAuth() && service?.status === 'shutdown'" class="dropdown-item"
+                          (click)="setService(service);openModal('confirmation-dialog')">
+                          <span>Enable</span>
+                        </a>
+                        <a *ngIf="sharedService.checkAuth()" class="dropdown-item"
+                          (click)="setService(service);openModal('delete-confirmation-dialog')">
+                          <span>Delete</span>
+                        </a>
+                      </div>
+                    </div>
+                  </div>
+                  <a *ngIf="!service?.name" (click)="openServiceModal(service?.type)" class="button is-text text-btn">
+                    {{service?.btntxt}}
+                  </a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+      </div>
+    </div>
+  </div>
+  <app-manage-service-modal (notify)="checkSelectedServiceStatus()" [serviceData]="serviceData"></app-manage-service-modal>
+  <app-confirmation-dialog id="delete-confirmation-dialog">
+    <header class="modal-card-head">
+      <p class="modal-card-title is-size-6">Delete</p>
+      <button type="button" class="delete" aria-label="close" (click)="closeModal('delete-confirmation-dialog')"></button>
+    </header>
+    <section class="modal-card-body">
+      Deleting this service can not be undone. Continue?
+    </section>
+    <footer class="modal-card-foot">
+      <button type="button" class="button is-small" (click)="closeModal('delete-confirmation-dialog')">Cancel</button>
+      <button type="button" class="button is-small is-danger" (click)="deleteService()" [appDisableUntilResponse]="reenableButton">Delete</button>
+    </footer>
+  </app-confirmation-dialog>
+  <app-confirmation-dialog id="confirmation-dialog">
+    <header class="modal-card-head">
+      <p class="modal-card-title is-size-6">{{service?.status === 'shutdown' ? 'Enable' : 'Disable'}}</p>
+      <button type="button" class="delete" aria-label="close" (click)="closeModal('confirmation-dialog')"></button>
+    </header>
+    <section class="modal-card-body">
+      Are you sure, You want to {{service?.status === 'shutdown' ? 'enable' : 'disable'}} the service {{service?.name}}?
+    </section>
+    <footer class="modal-card-foot">
+      <button type="button" class="button is-small" (click)="closeModal('confirmation-dialog')">Cancel</button>
+      <button type="button" class="button is-small" [ngClass]="{'is-info': service?.status === 'shutdown', 'is-danger': service?.status !== 'shutdown'}" (click)="stateUpdate()" [appDisableUntilResponse]="reenableButton">{{service?.status === 'shutdown' ? 'Enable' : 'Disable'}}</button>
```

Comment:
Use proper tag class for disabled/shutdown state. 

shutdown is not failed down (danger)

Reply:
These classes are for Enable and Disable buttons on confirm dialog that is appearing on clicking Enable/Disable action button from context menu.
`is-info` is for Enable button and `is-danger` is for Disable button.